velox/functions/sparksql/Arithmetic.h

```diff
@@ -351,6 +351,45 @@ struct ToHexBigintFunction {
   }
 };
 
+template <typename T>
+struct WidthBucketFunction {
+  FOLLY_ALWAYS_INLINE bool call(
+      int64_t& result,
+      double operand,
+      double bound1,
+      double bound2,
+      int64_t bucketCount) {
+    if (bucketCount <= 0 ||
+        bucketCount == std::numeric_limits<long long>::max() ||
+        std::isnan(operand) || bound1 == bound2 || std::isnan(bound1) ||
+        std::isinf(bound1) || std::isnan(bound2) || std::isinf(bound2)) {
+      return false;
+    }
+
+    double lower = std::min(bound1, bound2);
```

Comment:
`auto [lower, higher] = bound1 > bound2 ? std::make_pair(bound2, bound1) : std::make_pair(bound1, bound2);`

It can only compare once

Reply:
not need anymore since i refactor with how spark does